asmcomp/interval.ml

```diff
@@ -38,10 +38,11 @@ type kind =
   | Argument
   | Live
 
-let interval_list = ref ([] : t list)
-let fixed_interval_list = ref ([] : t list)
-let all_intervals() = !interval_list
-let all_fixed_intervals() = !fixed_interval_list
+type interval_result =
```

Comment:
Nitpicking, but perhaps we can call this type just "result" (since it already lives in a module called `Interval`)?